lib/std/sums.nim

```diff
@@ -53,6 +53,136 @@ func sumPairs*[T](x: openArray[T]): T =
   let n = len(x)
   if n == 0: T(0) else: sumPairwise(x, 0, n)
 
+func fastTwoSum*[T : SomeFloat](a, b: T): (T, T) =
```

Comment:
I would prefer naming the function using the Algorithm name, 
because this `fastThing()` will not scale, whats next `fastestThing()`, `soFastBroThing()`, `gottaGoFastThing()`.
:slightly_smiling_face:

Reply:
Unfortunately, it's the name of the algorithm. At least widely used :smile: 
For reference https://hal.archives-ouvertes.fr/hal-00164607/document ($2.2)